apps/src/templates/teacherDashboard/EditSectionForm.jsx

```diff
@@ -130,6 +139,49 @@ class EditSectionForm extends Component {
       isNewSection
     } = this.props;
 
+    /**
+    OAuth login types can not be changed.
+    Personal email login type can be changed to word OR picture login type.
+    Picture login type can be changed to word login type.
+    Word login type can be changed to picture login type.
+    **/
+    const changableLoginTypes = [
+      SectionLoginType.word,
+      SectionLoginType.picture,
+      SectionLoginType.email
+    ];
+
+    // Sections that do not yet have students can use any login type.
```

Comment:
I would find this comment a little bit more useful if it was directly next to line 157, to which it pertains. 

Better yet, I would find this comment even more useful if we could figure out a way for it to not need to exist.

The best comment is one that doesn't need to be written because the code clearly communicates the same  information that the comment does. Reading english comments can break the flow of reading a program, and also comments are often times innacurate or become innacurate with time as people edit a program, and as such it is best to use them sparingly and with caution. 


In light of the above opinion and supporting argument for it, how would you feel about it if the code at line 157 read something like this, with no explaining comment?

```
validLoginTypes = Object.values(SectionLoginType)
```